docs/docs/zh/guide/installation/standalone.md

```diff
@@ -27,7 +27,7 @@ bash ./bin/dolphinscheduler-daemon.sh start standalone-server
 
 ### 登录 DolphinScheduler
 
-浏览器访问地址 http://localhost:12345/dolphinscheduler/ui 即可登录系统UI。默认的用户名和密码是 **admin/dolphinscheduler123**
+浏览器访问地址 http://localhost:12345/dolphinscheduler/ui/ 即可登录系统UI。默认的用户名和密码是 **admin/dolphinscheduler123**
```

Comment:
@mklzl  Could u plz update the English docs to keep it consistent? Thx https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/guide/installation/standalone.md

Reply:
ok